### ISSUE TITLE
Not see entered value v-select with option 'autocomplete'

### ISSUE THREAD

**Issue by chim666** [vuetifyjs/vuetify#1398]
### Steps to reproduce
1. Reduce the window so that there is not enough space for the entire list from the bottom
1. Сlick to v-select 
2. Start input

### Versions
0.14.11

### What is expected ?
I see the value entered

### What is actually happening ?
<img src='https://i.imgur.com/tRJ3Dy1.gif' />
I do not see the value entered

### Reproduction Link
https://codepen.io/anon/pen/dzdGzW?editors=1010

**Comment by chancet1982** [comment]
This seems like a menu positioning issue: From investigating it myself yesterday. I found that the current positioning of the menu does not comply with how material design autocomplete shoudl work. I commented on the previous fix as it does not solve the problem. I also have an (incomplete) fix which currently disables the yOverflow check if autocomplete is "on" and ensures nudgeTop is zero when using autocomplete. this in itself is closer to the expected behavior but still does not comply with angular's own positioning as seen on:
https://material.angularjs.org/latest/demo/autocomplete

**TLDR:** 
1. textfield(select) should never be hidden be autocomplete results
2. if results overflow page bottom, autocomplete results should be shown ABOVE the textfield(select)
3. if no room is available above for showing the autocomplete results, the max-height should be adjusted to accomodate for the max number of tiles above the textfield(select)

**Comment by bpiaskiewicz** [comment]
This is also happening when I add 'hide-details' prop to component: https://codepen.io/anon/pen/rzJLPb

**Comment by johnleider** [comment]
This is something I started the foundation for last night, should be easy enough.

**Comment by johnleider** [comment]
This was fixed as part of #1408. Also made some great improvements to keyboard support and functionality for ```v-select``` with the **autocomplete** prop.

**Comment by flakekun** [comment]
Menu is still hovers above input if 'hide-details' attribute is used. Like in:
https://codepen.io/anon/pen/rzJLPb
